dnf-docker-test/features/steps/module_steps.py

```diff
@@ -7,7 +7,7 @@
 from file_steps import step_an_ini_file_filepath_should_contain
 import table_utils
 
-@then('a module "{modulename}" config file should contain')
+@then('a module {modulename} config file should contain')
```

Comment:
Thank you for the catch. However I am afraid that adding such change would break existing tests. Would you also add a piece of code that would remove the quotes if they are present to ensure the compatibility with existing tests?